torchqdynamics/utils/utils.py

```diff
@@ -173,70 +171,126 @@ def dissipator(L: Tensor, rho: Tensor) -> Tensor:
 def lindbladian(H: Tensor, Ls: Tensor, rho: Tensor) -> Tensor:
     r"""Apply the Lindbladian superoperator to a density matrix.
 
-    The system Lindbladian $\mathcal{L}(\cdot)$ is the superoperator
-    generating the evolution of the system. It is defined by
+    The Lindbladian superoperator $\mathcal{L}$ is defined by:
+
     $$
-        \frac{\mathrm{d}\rho}{\mathrm{d}t} = \mathcal{L}(\rho) = -i[H,\rho]
-                                             + \sum_{i=1}^n \mathcal{D}[L_i](\rho).
+        \mathcal{L}(\rho) = -i[H,\rho] + \sum_{k=1}^N \mathcal{D}[L_k](\rho),
     $$
 
+    where $H$ is the system Hamiltonian, $\{L_k\}$ is a set of $N$ jump operators
+    (arbitrary operators) and $\mathcal{D}[L]$ is the Lindblad dissipation superoperator
+    (see [dissipator()][torchqdynamics.dissipator]).
+
+    Note:
+        This superoperator is also sometimes called the *Liouvillian*.
```

Comment:
```suggestion
        This superoperator is also sometimes called *Liouvillian*.
```

Reply:
They are synonyms and used rather equally by different communities, I just thought if someone searches `Liouvillian` in the search bar he should end up here...